notebooks/blog_materials/Time_Series_Forecasting_with_Unstructured_Data_and_Amazon_SageMaker_Neural_Topic_Model/2_NTM.ipynb

```diff
@@ -830,7 +838,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.5"
+   "version": "3.7.3"
```

Comment:
This is not a critical problem, but could you use old kernel spec ("conda_python3") and Python version, so that
1. customers don't see "Kernel not found" dialog when this notebook is opened on the SageMaker Jupyter environment.
2. consistency with other notebooks.